AEPIntegrationTests/IdentityIntegrationTests.swift

```diff
@@ -70,6 +73,29 @@ class IdentityIntegrationTests: XCTestCase {
         wait(for: [requestExpectation], timeout: 1)
     }
 
+    /// No network request should be made since the Edge extension is registered
+    func testSyncIdentifiersEdgeIsRegistered() {
```

Comment:
Can we move this in a separate file, e.g. EdgeIdentityIntegrationTests? I expect we will add more integration tests for Edge use-cases later on.

Reply:
Similar comment here, we should have more e2e tests check the shared states are always up-to-date when public APIs are called, and that no network calls are going out.

Reply:
Moved to another file, regarding checking shared state... our integration tests currently don't check shared state so this is something we'd need to look into for future integration tests.